challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |word| word.capitalize }
```

Comment:
I checked the style guide for a reason and none is given: https://github.com/bbatsov/ruby-style-guide#single-action-blocks

It is shorter, obviously. I suppose it has become popular enough with rubyists to be expected by most.